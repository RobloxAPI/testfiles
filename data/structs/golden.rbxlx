<roblox version="4">
	<Meta name="MetaKeyOverride">MetaValueA</Meta>
	<Meta name="MetaKeyOverride">MetaValueB</Meta>
	<Item class="TestValues">
		<Properties>
			<string name="StringSimple">Hello world!</string>
			<string name="StringUnicode">Hello, 世界!</string>
			<string name="StringEscaped">Hello&#10;&#9;world!&#10;</string>
			<string name="StringBytes">Hello&#0;&#1;&#2;&#9;&#10;&#126;&#127;&#128;&#129;&#253;&#254;&#255;world!</string>
			<BinaryString name="BinaryStringSimple">hvsmnRkNLIX24EaM7KQqIA==</BinaryString>
			<BinaryString name="BinaryStringUnicode">MB4hSem/vmxZaqYx9pJGlw==</BinaryString>
			<BinaryString name="BinaryStringEscaped">0/eiDQu4pAxywAgOUW2cRA==</BinaryString>
			<BinaryString name="BinaryStringBytes">CnevubXmEHHrnmAN9Ghqow==</BinaryString>
			<ProtectedString name="ProtectedStringSimple">Hello world!</ProtectedString>
			<ProtectedString name="ProtectedStringUnicode">Hello, 世界!</ProtectedString>
			<ProtectedString name="ProtectedStringEscaped">Hello&#10;&#9;world!&#10;</ProtectedString>
			<ProtectedString name="ProtectedStringBytes">Hello&#0;&#1;&#2;&#9;&#10;&#126;&#127;&#128;&#129;&#253;&#254;&#255;world!</ProtectedString>
			<Content name="ContentNull"><null></null></Content>
			<Content name="ContentSimple"><url>Hello world!</url></Content>
			<Content name="ContentUnicode"><url>Hello, 世界!</url></Content>
			<Content name="ContentEscaped"><url>Hello&#10;&#9;world!&#10;</url></Content>
			<Content name="ContentBytes"><url>Hello&#0;&#1;&#2;&#9;&#10;&#126;&#127;&#128;&#129;&#253;&#254;&#255;world!</url></Content>
			<SharedString name="SharedStringSimple">hvsmnRkNLIX24EaM7KQqIA==</SharedString>
			<SharedString name="SharedStringUnicode">MB4hSem/vmxZaqYx9pJGlw==</SharedString>
			<SharedString name="SharedStringEscaped">0/eiDQu4pAxywAgOUW2cRA==</SharedString>
			<SharedString name="SharedStringBytes">CnevubXmEHHrnmAN9Ghqow==</SharedString>
			<bool name="BoolTrue">true</bool>
			<bool name="BoolFalse">false</bool>
			<float name="FloatGravity">196.2</float>
			<float name="FloatRound">99003.78125</float>
			<float name="FloatFive">-0.55555555555555555</float>
			<float name="FloatPi">3.14159274</float>
			<float name="FloatE">2.71828182845904523536028747135266249775724709369995957496696763</float>
			<float name="FloatPi">3.14159265358979323846264338327950288419716939937510582097494459</float>
			<float name="FloatPhi">1.61803398874989484820458683436563811772030917980576286213544862</float>
			<float name="FloatSqrt2">1.41421356237309504880168872420969807856967187537694807317667974</float>
			<float name="FloatSqrtE">1.64872127070012814684865078781416357165377610071014801157507931</float>
			<float name="FloatSqrtPi">1.77245385090551602729816748334114518279754945612238712821380779</float>
			<float name="FloatSqrtPhi">1.27201964951406896425242246173749149171560804184009624861664038</float>
			<float name="FloatLn2">0.693147180559945309417232121458176568075500134360255254120680009</float>
			<float name="FloatLn10">2.30258509299404568401799145468436420760110148862877297603332790</float>
			<double name="DoubleGravity">196.2</double>
			<double name="DoubleRound">99003.78125</double>
			<double name="DoubleFive">-0.55555555555555555</double>
			<double name="DoublePi">3.14159274</double>
			<double name="DoubleE">2.71828182845904523536028747135266249775724709369995957496696763</double>
			<double name="DoublePi">3.14159265358979323846264338327950288419716939937510582097494459</double>
			<double name="DoublePhi">1.61803398874989484820458683436563811772030917980576286213544862</double>
			<double name="DoubleSqrt2">1.41421356237309504880168872420969807856967187537694807317667974</double>
			<double name="DoubleSqrtE">1.64872127070012814684865078781416357165377610071014801157507931</double>
			<double name="DoubleSqrtPi">1.77245385090551602729816748334114518279754945612238712821380779</double>
			<double name="DoubleSqrtPhi">1.27201964951406896425242246173749149171560804184009624861664038</double>
			<double name="DoubleLn2">0.693147180559945309417232121458176568075500134360255254120680009</double>
			<double name="DoubleLn10">2.30258509299404568401799145468436420760110148862877297603332790</double>
			<int name="IntMinUnderflowMore">-2147483650</int>
			<int name="IntMinUnderflow">-2147483649</int>
			<int name="IntMin">-2147483648</int>
			<int name="IntMinIn">-2147483647</int>
			<int name="IntMinusTwo">-2</int>
			<int name="IntMinusOne">-1</int>
			<int name="IntZero">0</int>
			<int name="IntOne">1</int>
			<int name="IntMaxIn">2147483646</int>
			<int name="IntMax">2147483647</int>
			<int name="IntMaxOverflow">2147483648</int>
			<int name="IntMaxOverflowMore">2147483649</int>
			<int64 name="Int64MinUnderflowMore">-9223372036854775810</int64>
			<int64 name="Int64MinUnderflow">-9223372036854775809</int64>
			<int64 name="Int64Min">-9223372036854775808</int64>
			<int64 name="Int64MinIn">-9223372036854775807</int64>
			<int64 name="Int64MinusTwo">-2</int64>
			<int64 name="Int64MinusOne">-1</int64>
			<int64 name="Int64Zero">0</int64>
			<int64 name="Int64One">1</int64>
			<int64 name="Int64MaxIn">9223372036854775806</int64>
			<int64 name="Int64Max">9223372036854775807</int64>
			<int64 name="Int64MaxOverflow">9223372036854775808</int64>
			<int64 name="Int64MaxOverflowMore">9223372036854775809</int64>
			<UDim name="UDim">
				<S>99000.78125</S>
				<O>1234567890</O>
			</UDim>
			<UDim2 name="UDim2">
				<XS>99000.78125</S>
				<XO>1234567890</O>
				<YS>-99001.78125</S>
				<YO>-1234567891</O>
			</UDim>
			<Ray name="Ray">
				<origin>
					<X>99000.78125</X>
					<Y>-99001.78125</Y>
					<Z>99002.78125</Z>
				</origin>
				<direction>
					<X>-99003.78125</X>
					<Y>99004.78125</Y>
					<Z>-99005.78125</Z>
				</direction>
			</Ray>
			<Faces name="Faces"><faces>21</faces></Faces>
			<Axes name="Axes"><axes>6</axes></Axes>
			<BrickColor name="BrickColor">1234567890</BrickColor>
			<Color3 name="Color3">
				<R>99000.78125</R>
				<G>-99001.78125</G>
				<B>99002.78125</B>
			</Color3>
			<Vector2 name="Vector2">
				<X>99000.78125</X>
				<Y>-99001.78125</Y>
			</Vector2>
			<Vector3 name="Vector3">
				<X>99000.78125</X>
				<Y>-99001.78125</Y>
				<Z>99002.78125</Z>
			</Vector3>
			<CFrame name="CFrame">
				<X>99000.78125</X>
				<Y>-99001.78125</Y>
				<Z>99002.78125</Z>
				<R00>-99003.78125</R00>
				<R01>99004.78125</R00>
				<R02>-99005.78125</R00>
				<R10>99006.78125</R00>
				<R11>-99007.78125</R00>
				<R12>99008.78125</R00>
				<R20>-99009.78125</R00>
				<R21>99010.78125</R00>
				<R22>-99011.78125</R00>
			</CFrame>
			<token name="Token">1234567890</token>
			<NumberSequence name="NumberSequence">
				99000.78125 -99001.78125 99002.78125
				-99003.78125 99004.78125 -99005.78125
				99006.78125 -99007.78125 99008.78125
				-99009.78125 99010.78125 -99011.78125
			</NumberSequence>
			<ColorSequence name="ColorSequence">
				99000.78125 -99001.78125 99002.78125 -99003.78125 99004.78125
				-99005.78125 99006.78125 -99007.78125 99008.78125 -99009.78125
				99010.78125 -99011.78125 99012.78125 -99013.78125 99014.78125
			</ColorSequence>
			<NumberRange name="NumberRange">99000.78125 -99001.78125 </NumberRange>
			<Rect2D name="Rect2D">
				<min>
					<X>99000.78125</X>
					<Y>-99001.78125</Y>
				</min>
				<max>
					<X>99002.78125</X>
					<Y>-99003.78125</Y>
				</max>
			</Rect2D>
			<PhysicalProperties name="PhysicalPropertiesFalseEmpty">
				<CustomPhysics>false</CustomPhysics>
			</PhysicalProperties>
			<PhysicalProperties name="PhysicalPropertiesFalse">
				<CustomPhysics>false</CustomPhysics>
				<Density>99000.78125</Density>
				<Friction>-99001.78125</Friction>
				<Elasticity>99002.78125</Elasticity>
				<FrictionWeight>-99003.78125</FrictionWeight>
				<ElasticityWeight>99004.78125</ElasticityWeight>
			</PhysicalProperties>
			<PhysicalProperties name="PhysicalPropertiesTrueEmpty">
				<CustomPhysics>true</CustomPhysics>
			</PhysicalProperties>
			<PhysicalProperties name="PhysicalPropertiesTrue">
				<CustomPhysics>true</CustomPhysics>
				<Density>99000.78125</Density>
				<Friction>-99001.78125</Friction>
				<Elasticity>99002.78125</Elasticity>
				<FrictionWeight>-99003.78125</FrictionWeight>
				<ElasticityWeight>99004.78125</ElasticityWeight>
			</PhysicalProperties>
			<Color3uint8 name="Color3uint8">
				<R>1234567890</R>
				<G>-1234567891</G>
				<B>1234567892</B>
			</Color3uint8>
		</Properties>
	</Item>
	<SharedStrings>
		<SharedString md5="hvsmnRkNLIX24EaM7KQqIA==">SGVsbG8gd29ybGQh</SharedString>
		<SharedString md5="MB4hSem/vmxZaqYx9pJGlw==">SGVsbG8sIOS4lueVjCE=</SharedString>
		<SharedString md5="0/eiDQu4pAxywAgOUW2cRA==">SGVsbG8KCXdvcmxkIQo=</SharedString>
		<SharedString md5="CnevubXmEHHrnmAN9Ghqow==">SGVsbG8AAQIJCn5/gIH9/v93b3JsZCE=</SharedString>
	</SharedStrings>
	<Item class="ReferentA" referent="aaa"></Item>
	<Meta name="MetaKeyMiddle">MetaValueMiddle</Meta>
	<Meta name="MetaASort">MetaValueSort</Meta>
	<Item class="RefererA">
		<Properties>
			<Ref name="ReferenceA">aaa</Ref>
		</Properties>
	</Item>
	<Item class="RefererB">
		<Properties>
			<Ref name="ReferenceB">bbb</Ref>
		</Properties>
	</Item>
	<Item class="ReferentB" referent="bbb"></Item>
</roblox>
