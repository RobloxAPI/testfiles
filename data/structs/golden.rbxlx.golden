{
	"Format": "rbxlx",
	"Output": "model",
	"Data": {
		"Metadata": [
			{
				"Key": "MetaASort",
				"Value": "MetaValueSort"
			},
			{
				"Key": "MetaKeyMiddle",
				"Value": "MetaValueMiddle"
			},
			{
				"Key": "MetaKeyOverride",
				"Value": "MetaValueB"
			}
		],
		"Instances": [
			{
				"ClassName": "TestValues",
				"IsService": false,
				"Reference": 0,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": true,
							"Z": true
						}
					},
					{
						"Name": "BoolFalse",
						"Type": "Bool",
						"Value": false
					},
					{
						"Name": "BoolTrue",
						"Type": "Bool",
						"Value": true
					},
					{
						"Name": "BrickColor",
						"Type": "BrickColor",
						"Value": 1234567890
					},
					{
						"Name": "CFrame",
						"Type": "CFrame",
						"Value": {
							"Position": {
								"X": 99000.7812,
								"Y": -99001.7812,
								"Z": 99002.7812
							},
							"Rotation": {
								"R00": -99003.7812,
								"R01": 99004.7812,
								"R02": -99005.7812,
								"R10": 99006.7812,
								"R11": -99007.7812,
								"R12": 99008.7812,
								"R20": -99009.7812,
								"R21": 99010.7812,
								"R22": -99011.7812
							}
						}
					},
					{
						"Name": "Color3",
						"Type": "Color3",
						"Value": {
							"R": 99000.7812,
							"G": -99001.7812,
							"B": 99002.7812
						}
					},
					{
						"Name": "Color3uint8",
						"Type": "Color3uint8",
						"Value": {
							"R": 210,
							"G": 45,
							"B": 212
						}
					},
					{
						"Name": "ColorSequence",
						"Type": "ColorSequence",
						"Value": [
							{
								"Time": 99000.7812,
								"Value": {
									"R": -99001.7812,
									"G": 99002.7812,
									"B": -99003.7812
								},
								"Envelope": 99004.7812
							},
							{
								"Time": -99005.7812,
								"Value": {
									"R": 99006.7812,
									"G": -99007.7812,
									"B": 99008.7812
								},
								"Envelope": -99009.7812
							},
							{
								"Time": 99010.7812,
								"Value": {
									"R": -99011.7812,
									"G": 99012.7812,
									"B": -99013.7812
								},
								"Envelope": 99014.7812
							}
						]
					},
					{
						"Name": "ContentBytes",
						"Type": "Content",
						"Value": [
							 72,101,108,108,111,  0,  1,  2,  9, 10,126,127,128,129,253,254,
							255,119,111,114,108,100, 33
						]
					},
					{
						"Name": "ContentEscaped",
						"Type": "Content",
						"Value": "Hello\n\tworld!\n"
					},
					{
						"Name": "ContentNull",
						"Type": "Content",
						"Value": ""
					},
					{
						"Name": "ContentSimple",
						"Type": "Content",
						"Value": "Hello world!"
					},
					{
						"Name": "ContentUnicode",
						"Type": "Content",
						"Value": "Hello, 世界!"
					},
					{
						"Name": "DoubleE",
						"Type": "Double",
						"Value": 2.7182818284590451
					},
					{
						"Name": "DoubleFive",
						"Type": "Double",
						"Value": -0.55555555555555558
					},
					{
						"Name": "DoubleGravity",
						"Type": "Double",
						"Value": 196.19999999999999
					},
					{
						"Name": "DoubleLn10",
						"Type": "Double",
						"Value": 2.3025850929940459
					},
					{
						"Name": "DoubleLn2",
						"Type": "Double",
						"Value": 0.69314718055994529
					},
					{
						"Name": "DoublePhi",
						"Type": "Double",
						"Value": 1.6180339887498949
					},
					{
						"Name": "DoublePi",
						"Type": "Double",
						"Value": 3.1415926535897931
					},
					{
						"Name": "DoubleRound",
						"Type": "Double",
						"Value": 99003.78125
					},
					{
						"Name": "DoubleSqrt2",
						"Type": "Double",
						"Value": 1.4142135623730951
					},
					{
						"Name": "DoubleSqrtE",
						"Type": "Double",
						"Value": 1.6487212707001282
					},
					{
						"Name": "DoubleSqrtPhi",
						"Type": "Double",
						"Value": 1.272019649514069
					},
					{
						"Name": "DoubleSqrtPi",
						"Type": "Double",
						"Value": 1.7724538509055161
					},
					{
						"Name": "Faces",
						"Type": "Faces",
						"Value": {
							"Right": true,
							"Top": false,
							"Back": true,
							"Left": false,
							"Bottom": true,
							"Front": false
						}
					},
					{
						"Name": "FloatE",
						"Type": "Float",
						"Value": 2.71828175
					},
					{
						"Name": "FloatFive",
						"Type": "Float",
						"Value": -0.555555582
					},
					{
						"Name": "FloatGravity",
						"Type": "Float",
						"Value": 196.199997
					},
					{
						"Name": "FloatLn10",
						"Type": "Float",
						"Value": 2.30258512
					},
					{
						"Name": "FloatLn2",
						"Type": "Float",
						"Value": 0.693147182
					},
					{
						"Name": "FloatPhi",
						"Type": "Float",
						"Value": 1.61803401
					},
					{
						"Name": "FloatPi",
						"Type": "Float",
						"Value": 3.14159274
					},
					{
						"Name": "FloatRound",
						"Type": "Float",
						"Value": 99003.7812
					},
					{
						"Name": "FloatSqrt2",
						"Type": "Float",
						"Value": 1.41421354
					},
					{
						"Name": "FloatSqrtE",
						"Type": "Float",
						"Value": 1.64872122
					},
					{
						"Name": "FloatSqrtPhi",
						"Type": "Float",
						"Value": 1.27201962
					},
					{
						"Name": "FloatSqrtPi",
						"Type": "Float",
						"Value": 1.7724539
					},
					{
						"Name": "Int64Max",
						"Type": "Int64",
						"Value": 9223372036854775807
					},
					{
						"Name": "Int64MaxIn",
						"Type": "Int64",
						"Value": 9223372036854775806
					},
					{
						"Name": "Int64MaxOverflow",
						"Type": "Int64",
						"Value": 9223372036854775807
					},
					{
						"Name": "Int64MaxOverflowMore",
						"Type": "Int64",
						"Value": 9223372036854775807
					},
					{
						"Name": "Int64Min",
						"Type": "Int64",
						"Value": -9223372036854775808
					},
					{
						"Name": "Int64MinIn",
						"Type": "Int64",
						"Value": -9223372036854775807
					},
					{
						"Name": "Int64MinUnderflow",
						"Type": "Int64",
						"Value": -9223372036854775808
					},
					{
						"Name": "Int64MinUnderflowMore",
						"Type": "Int64",
						"Value": -9223372036854775808
					},
					{
						"Name": "Int64MinusOne",
						"Type": "Int64",
						"Value": -1
					},
					{
						"Name": "Int64MinusTwo",
						"Type": "Int64",
						"Value": -2
					},
					{
						"Name": "Int64One",
						"Type": "Int64",
						"Value": 1
					},
					{
						"Name": "Int64Zero",
						"Type": "Int64",
						"Value": 0
					},
					{
						"Name": "IntMax",
						"Type": "Int",
						"Value": 2147483647
					},
					{
						"Name": "IntMaxIn",
						"Type": "Int",
						"Value": 2147483646
					},
					{
						"Name": "IntMaxOverflow",
						"Type": "Int",
						"Value": 2147483647
					},
					{
						"Name": "IntMaxOverflowMore",
						"Type": "Int",
						"Value": 2147483647
					},
					{
						"Name": "IntMin",
						"Type": "Int",
						"Value": -2147483648
					},
					{
						"Name": "IntMinIn",
						"Type": "Int",
						"Value": -2147483647
					},
					{
						"Name": "IntMinUnderflow",
						"Type": "Int",
						"Value": -2147483648
					},
					{
						"Name": "IntMinUnderflowMore",
						"Type": "Int",
						"Value": -2147483648
					},
					{
						"Name": "IntMinusOne",
						"Type": "Int",
						"Value": -1
					},
					{
						"Name": "IntMinusTwo",
						"Type": "Int",
						"Value": -2
					},
					{
						"Name": "IntOne",
						"Type": "Int",
						"Value": 1
					},
					{
						"Name": "IntZero",
						"Type": "Int",
						"Value": 0
					},
					{
						"Name": "NumberRange",
						"Type": "NumberRange",
						"Value": {
							"Min": 99000.7812,
							"Max": -99001.7812
						}
					},
					{
						"Name": "NumberSequence",
						"Type": "NumberSequence",
						"Value": [
							{
								"Time": 99000.7812,
								"Value": -99001.7812,
								"Envelope": 99002.7812
							},
							{
								"Time": -99003.7812,
								"Value": 99004.7812,
								"Envelope": -99005.7812
							},
							{
								"Time": 99006.7812,
								"Value": -99007.7812,
								"Envelope": 99008.7812
							},
							{
								"Time": -99009.7812,
								"Value": 99010.7812,
								"Envelope": -99011.7812
							}
						]
					},
					{
						"Name": "PhysicalPropertiesFalse",
						"Type": "PhysicalProperties",
						"Value": {
							"CustomPhysics": false,
							"Density": null,
							"Friction": null,
							"Elasticity": null,
							"FrictionWeight": null,
							"ElasticityWeight": null
						}
					},
					{
						"Name": "PhysicalPropertiesFalseEmpty",
						"Type": "PhysicalProperties",
						"Value": {
							"CustomPhysics": false,
							"Density": null,
							"Friction": null,
							"Elasticity": null,
							"FrictionWeight": null,
							"ElasticityWeight": null
						}
					},
					{
						"Name": "PhysicalPropertiesTrue",
						"Type": "PhysicalProperties",
						"Value": {
							"CustomPhysics": true,
							"Density": 99000.7812,
							"Friction": -99001.7812,
							"Elasticity": 99002.7812,
							"FrictionWeight": -99003.7812,
							"ElasticityWeight": 99004.7812
						}
					},
					{
						"Name": "PhysicalPropertiesTrueEmpty",
						"Type": "PhysicalProperties",
						"Value": {
							"CustomPhysics": true,
							"Density": 0,
							"Friction": 0,
							"Elasticity": 0,
							"FrictionWeight": 0,
							"ElasticityWeight": 0
						}
					},
					{
						"Name": "ProtectedStringBytes",
						"Type": "ProtectedString",
						"Value": [
							 72,101,108,108,111,  0,  1,  2,  9, 10,126,127,128,129,253,254,
							255,119,111,114,108,100, 33
						]
					},
					{
						"Name": "ProtectedStringEscaped",
						"Type": "ProtectedString",
						"Value": "Hello\n\tworld!\n"
					},
					{
						"Name": "ProtectedStringSimple",
						"Type": "ProtectedString",
						"Value": "Hello world!"
					},
					{
						"Name": "ProtectedStringUnicode",
						"Type": "ProtectedString",
						"Value": "Hello, 世界!"
					},
					{
						"Name": "Ray",
						"Type": "Ray",
						"Value": {
							"Origin": {
								"X": 99000.7812,
								"Y": -99001.7812,
								"Z": 99002.7812
							},
							"Direction": {
								"X": -99003.7812,
								"Y": 99004.7812,
								"Z": -99005.7812
							}
						}
					},
					{
						"Name": "Rect2D",
						"Type": "Rect2D",
						"Value": {
							"Min": {
								"X": 99000.7812,
								"Y": -99001.7812
							},
							"Max": {
								"X": 99002.7812,
								"Y": -99003.7812
							}
						}
					},
					{
						"Name": "SharedStringBytes",
						"Type": "SharedString",
						"Value": [
							 72,101,108,108,111,  0,  1,  2,  9, 10,126,127,128,129,253,254,
							255,119,111,114,108,100, 33
						]
					},
					{
						"Name": "SharedStringEscaped",
						"Type": "SharedString",
						"Value": "Hello\n\tworld!\n"
					},
					{
						"Name": "SharedStringSimple",
						"Type": "SharedString",
						"Value": "Hello world!"
					},
					{
						"Name": "SharedStringUnicode",
						"Type": "SharedString",
						"Value": "Hello, 世界!"
					},
					{
						"Name": "StringBytes",
						"Type": "String",
						"Value": [
							 72,101,108,108,111,  0,  1,  2,  9, 10,126,127,128,129,253,254,
							255,119,111,114,108,100, 33
						]
					},
					{
						"Name": "StringEscaped",
						"Type": "String",
						"Value": "Hello\n\tworld!\n"
					},
					{
						"Name": "StringSimple",
						"Type": "String",
						"Value": "Hello world!"
					},
					{
						"Name": "StringUnicode",
						"Type": "String",
						"Value": "Hello, 世界!"
					},
					{
						"Name": "Token",
						"Type": "Token",
						"Value": 1234567890
					},
					{
						"Name": "UDim",
						"Type": "UDim",
						"Value": {
							"Scale": 99000.7812,
							"Offset": 1234567890
						}
					},
					{
						"Name": "UDim2",
						"Type": "UDim2",
						"Value": {
							"X": {
								"Scale": 99000.7812,
								"Offset": 1234567890
							},
							"Y": {
								"Scale": -99001.7812,
								"Offset": -1234567891
							}
						}
					},
					{
						"Name": "Vector2",
						"Type": "Vector2",
						"Value": {
							"X": 99000.7812,
							"Y": -99001.7812
						}
					},
					{
						"Name": "Vector3",
						"Type": "Vector3",
						"Value": {
							"X": 99000.7812,
							"Y": -99001.7812,
							"Z": 99002.7812
						}
					}
				],
				"Children": []
			},
			{
				"ClassName": "ReferentA",
				"IsService": false,
				"Reference": 1,
				"Properties": [],
				"Children": []
			},
			{
				"ClassName": "RefererA",
				"IsService": false,
				"Reference": 2,
				"Properties": [
					{
						"Name": "ReferenceA",
						"Type": "Reference",
						"Value": 1
					}
				],
				"Children": []
			},
			{
				"ClassName": "RefererB",
				"IsService": false,
				"Reference": 3,
				"Properties": [
					{
						"Name": "ReferenceB",
						"Type": "Reference",
						"Value": 4
					}
				],
				"Children": []
			},
			{
				"ClassName": "ReferentB",
				"IsService": false,
				"Reference": 4,
				"Properties": [],
				"Children": []
			}
		]
	}
}